config/crd/bases/cluster.x-k8s.io_machines.yaml

```diff
@@ -254,7 +254,8 @@ spec:
           - infrastructureRef
           type: object
         status:
-          description: MachineStatus defines the observed state of Machine
+          description: / [MachineStatus] MachineStatus defines the observed state
```

Comment:
This generation looks a bit off.

Reply:
This has been shown before as well, we were missing marks on the resources, I added them back given that we'll probably reference these from the gitbook